### DISCUSSION TITLE
Account flagged

### DISCUSSION THREAD

**Discussion by rashmisridar** [community/community#32191] [Other Feature Feedback, Questions, & Ideas]
My account has been flagged how to resolve it. 
![Git_account_flagged](https://user-images.githubusercontent.com/8536749/188812423-e3609b0f-c79c-4a79-a21f-bf93bd4f6f13.png)

**Comment by melvincwng** [comment]
@rashmisridar, Hi Rashmi 👋, for account related issues please contact [GitHub Support](https://support.github.com/contact/account) directly to resolve the issue.

Hope this clarifies and answers your question 😊

**Reply by rashmisridar** [reply]
Raised ticker over there but they have declined it

**Reply by melvincwng** [reply]
@rashmisridar Based on GitHub's Terms of Service, a person is only allowed to register for 1 GitHub account.

Having multiple GitHub accounts is not allowed actually.

Maybe that's why they declined your ticket?

**Comment by ghost** [comment]
I have same issue, i used business email to create account for my company, but my account has been flagged :(
